examples/federated/webpack.config.js

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+const data = require('./package.json');
+const Build = require('@jupyterlab/buildutils').Build;
+const webpack = require('webpack');
+const ModuleFederationPlugin = require('webpack/lib/container/ModuleFederationPlugin');
+const ContainerPlugin = require('webpack/lib/container/ContainerPlugin');
```

Comment:
You can now destructure off webpack.container

Reply:
Thanks, we'll do some cleanup once we get this all wired up.

Reply:
Done now. Thanks.